docs/examples.rst

```diff
@@ -4,6 +4,18 @@
 Examples
 ========
 
+To run the examples, download a full
+`test dataset <https://github.com/bihealth/altamisa/tree/master/tests/data/BII-I-1>`_
+(all files) from the AltamISA repository to your working directory.
+
+Import AltamISA (if not already done) and other required modules.
+
+.. code-block:: python
+
+    from altamisa.isatab import *
```

Comment:
This looks unfinished.

Reply:
Ah, now I see. Can we put this example into a file and include fragments of this file here? Sphinx allows to include file ranges.